service/matching/physical_task_queue_manager_interface.go

```diff
@@ -73,5 +73,8 @@ type (
 		// ShouldEmitGauges determines whether the gauge metrics should be emitted or not for this particular physical
 		// queue based on dynamic configs.
 		ShouldEmitGauges() bool
+		// MakePollerScalingDecision makes a decision on whether to scale pollers up or down based on the current state
+		// of the task queue and the task about to be returned. Does not modify inputs.
```

Comment:
`time.Time` is immutable, no need to say this
```suggestion
		// of the task queue and the task about to be returned.
```